python/beeai_framework/adapters/litellm/chat.py

```diff
@@ -181,13 +181,15 @@ def _transform_output(self, chunk: ModelResponse | ModelResponseStream) -> ChatM
                         AssistantMessage(
                             [
                                 MessageToolCallContent(
-                                    id=call.id or "dummy_id", tool_name=call.function.name, args=call.function.arguments
+                                    id=call.id or "dummy_id",
+                                    tool_name=call.function.name or "",
+                                    args=call.function.arguments,
                                 )
                                 for call in update.tool_calls
                             ]
                         )
                         if update.tool_calls
-                        else AssistantMessage(update.content)
+                        else AssistantMessage(str(update.content))
```

Comment:
This is not correct.

Reply:
I've reverted this and made it an ignore line instead

Reply:
I don't see any update.